src/Gizmos/gizmoManager.ts

```diff
@@ -71,6 +71,20 @@ export class GizmoManager implements IDisposable {
         return this._defaultUtilityLayer;
     }
 
+    /**
+     * True when the mouse pointer is hovering a gizmo mesh
+     */
+    public get isHovered() {
+        var hovered = false;
+        for (var key in this.gizmos) {
+            var gizmo = <Nullable<Gizmo>>((<any>this.gizmos)[key]);
+            if (gizmo) {
+                hovered = hovered || gizmo.isHovered;
```

Comment:
Nit: you could break here if you want!

Reply:
I get 'Jump target cannot cross function boundary.' when trying to break. Do you have some code suggestion on the proper way to do it?

Reply:
pinging @Popov72

Reply:
I don't know, something like that works for me:
```typescript
        var toto: { [name: string] : number } = { a: 1, b: 2, c: 3 };
        for (var k in toto) {
            if (toto[k] === 2) {
                break;
            }
        }
```

Reply:
Fixed. Thank you @Popov72